feat(module:select): add nzOnClear callback (#9188)
NG-ZORRO/ng-zorro-antd: components/select/doc/index.en-US.md

| `(nzScrollToBottom)`           | Called when dropdown scrolls to bottom                                                                                                                                                                          | `EventEmitter<any>`                                                                                                                                                       | -                                                       |
| `(nzOnSearch)`                 | Callback function that is fired when input changed.                                                                                                                                                             | `EventEmitter<string>`                                                                                                                                                    | -                                                       |

> | `(nzOnClear)`                  | Callback function that is fired when input is changed.                                                                                                                                                          | `EventEmitter<any>`                                                                                                                                                       | -                                                       |

Reviewer: ```suggestion
| `(nzOnClear)`                  | Callback function that clear the selected items                                                                                                                                                          | `EventEmitter<any>`                                                                                                                                                       | -                                                       |
```
